### ISSUE TITLE
[Bug]: TPCH 1G query results are incorrect and different nearly for every time when mo running on eks with 2 cn

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#7678]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):nightly-55b4d20b(imageid)
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

when mo running on eks with 2 cns, run tpch 1g 22 queries, nearly every turn, there were some queries result which are not correct.

for example:

q5 q6 not correct for this turn:

2023-01-17 11:28:43 Now start to compare query result for [report/TPCH_1/q5.txt,golden/TPCH_1/q5.txt]
2023-01-17 11:28:43 The column[0] value of row[1] does not equal to each other, [CANADA] and [PERU]
2023-01-17 11:28:43 The column[1] value of row[1] does not equal to each other, [51851090.643400000000] and [56206762.5035]
2023-01-17 11:28:43 The column[0] value of row[2] does not equal to each other, [PERU] and [CANADA]
2023-01-17 11:28:43 The column[1] value of row[2] does not equal to each other, [51224704.221700000000] and [56052846.0161]
2023-01-17 11:28:43 The column[1] value of row[3] does not equal to each other, [49619966.330100000000] and [54595012.8076]
2023-01-17 11:28:43 The column[1] value of row[4] does not equal to each other, [48953788.537500000000] and [53601776.5201]
2023-01-17 11:28:43 The column[1] value of row[5] does not equal to each other, [46699539.869200000000] and [50890580.8962]
2023-01-17 11:28:43 The query q5  has failed to  be executed,and cost: 339

2023-01-17 11:28:43 Now start to execute the query queries/q6.sql,please wait.....

2023-01-17 11:28:44 Now start to compare query result for [report/TPCH_1/q6.txt,golden/TPCH_1/q6.txt]
2023-01-17 11:28:44 The column[0] value of row[1] does not equal to each other, [30814775.0259] and [61660051.7967]
2023-01-17 11:28:44 The query q6  has failed to  be executed,and cost: 436

q4 q5 not correct for this turn:

2023-01-17 11:28:19 Now start to compare query result for [report/TPCH_1/q4.txt,golden/TPCH_1/q4.txt]
2023-01-17 11:28:19 The column[1] value of row[1] does not equal to each other, [7912] and [10623]
2023-01-17 11:28:19 The column[1] value of row[2] does not equal to each other, [7748] and [10465]
2023-01-17 11:28:19 The column[1] value of row[3] does not equal to each other, [7703] and [10309]
2023-01-17 11:28:19 The column[1] value of row[4] does not equal to each other, [7944] and [10618]
2023-01-17 11:28:19 The column[1] value of row[5] does not equal to each other, [7902] and [10541]
2023-01-17 11:28:19 The query q4  has failed to  be executed,and cost: 597

2023-01-17 11:28:19 Now start to execute the query queries/q5.sql,please wait.....

2023-01-17 11:28:19 Now start to compare query result for [report/TPCH_1/q5.txt,golden/TPCH_1/q5.txt]
2023-01-17 11:28:19 The column[1] value of row[1] does not equal to each other, [50243827.357400000000] and [56206762.5035]
2023-01-17 11:28:19 The column[1] value of row[2] does not equal to each other, [49229861.244300000000] and [56052846.0161]
2023-01-17 11:28:19 The column[0] value of row[3] does not equal to each other, [BRAZIL] and [ARGENTINA]
2023-01-17 11:28:19 The column[1] value of row[3] does not equal to each other, [48377073.716100000000] and [54595012.8076]
2023-01-17 11:28:19 The column[0] value of row[4] does not equal to each other, [ARGENTINA] and [BRAZIL]
2023-01-17 11:28:19 The column[1] value of row[4] does not equal to each other, [48348012.216300000000] and [53601776.5201]
2023-01-17 11:28:19 The column[1] value of row[5] does not equal to each other, [44688636.501000000000] and [50890580.8962]
2023-01-17 11:28:19 The query q5  has failed to  be executed,and cost: 387

**Comment by nnsgmsone** [comment]
This reason is caused by the presence of some disorder in the messages on the stream, wait and see what the situation is after consulting with Zhang Xu.

**Comment by aressu1985** [comment]
fixed: aca93daf2528a20ac54dbc91692df33131279f3e
